packages/mobile/src/screens/audio-screen/Tier.tsx

```diff
@@ -0,0 +1,142 @@
+import { ReactNode } from 'react'
+
+import { View } from 'react-native'
+import { Shadow } from 'react-native-shadow-2'
+
+import IconArrow from 'app/assets/images/iconArrow.svg'
+import { GradientText, Text } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+import { spacing } from 'app/styles/spacing'
+import { useThemeColors } from 'app/utils/theme'
+
+const messages = {
+  tier: 'Tier',
+  minAmount: '$AUDIO',
+  current: 'Current Tier'
+}
+
+const useStyles = makeStyles(({ spacing, palette, typography }) => ({
+  root: {
+    width: '100%',
+    marginVertical: spacing(8)
+  },
+  container: {
+    width: '100%',
+    justifyContent: 'center',
+    alignItems: 'center',
+    paddingVertical: spacing(10),
+    paddingHorizontal: spacing(4),
+    borderColor: palette.neutralLight7,
+    borderWidth: 2,
+    borderRadius: spacing(4),
+    backgroundColor: palette.neutralLight10,
+    flexDirection: 'column'
```

Comment:
Column is the default flex direction in RN

Reply:
ty

Reply:
well we have the h1,h2,h3... styles, which over time, we should figure out just how to keep 1-1 w/ julian's design so we're never off cannon

Reply:
yeah totally, and just a headsup im totally guessing on the existing values, so we should all def work to define them more rigorously